PersonalTwitterFeed/src/personaltwitterfeed/PersonalTwitterFeed.java

```diff
@@ -20,21 +20,27 @@
     /**
      * @param args the command line arguments
      */
-    public static void main(String[] args) {
-        String[] tweets = new String[MAX_NUMBER_TWEETS];
-        Scanner keyboard = new Scanner(System.in);
-        System.out.println("Welcome to your personal Twitter!");
-        System.out.println("What's your name, tweeter?");
+    
+    public static  String getCurrentTimeStamp(){
         
-        String tweeterName = keyboard.nextLine();
+        String pattern = "yyyy-MM-dd HH:mm:ss";
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat(pattern);
+                
+        String stamp = simpleDateFormat.format(new Date());
+        return stamp;
+    }
+    
+    public static void newTweet(String tweeterName){
+        
+         String[] tweets = new String[MAX_NUMBER_TWEETS];
+        Scanner keyboard = new Scanner(System.in);
+       
         
-        System.out.println("Nice to meet you " + tweeterName + "!");
-        System.out.println("Enter your tweets and I will add them to your timeline!");
         
         int numTweets = 0;
         
         while(numTweets < (MAX_NUMBER_TWEETS - 1)) {
-            tweets[numTweets] = keyboard.nextLine();
+            tweets[numTweets] = keyboard.nextLine()+" "+getCurrentTimeStamp();
```

Comment:
Great addition. You made it easy for us to see where you added the timestamp. Clear and concise. Solid method.